akeneo-design-system/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,116 @@
+import React, {FC} from 'react';
+import styled from 'styled-components';
+import {PAGINATION_SEPARATOR, PaginationItem} from './PaginationItem';
+
+type PaginationProps = {
+  /**
+   * The current page number (starting at 1).
+   */
+  currentPage: number;
+
+  /**
+   * The total number of results.
+   */
+  totalItems: number;
+
+  /**
+   * The number of items per page. Default to 25.
+   */
+  itemsPerPage?: number;
+
+  /**
+   * Handler called when a pagination item is clicked.
+   */
+  onClick: (page: number) => void;
```

Comment:
What do you think about replacing `onClick` that is a very technical action by a more functional like bellow:
```suggestion
  followPage: (page: number) => void;
```